libs/editor/WordPressEditor/src/main/res/layout/fragment_aztec_editor.xml

```diff
@@ -87,35 +84,36 @@
                     aztec:drawableLoading="@drawable/ic_image_loading_grey_a_40_48dp"
                     aztec:historyEnable="true"
                     aztec:historySize="10"
-                    aztec:linkColor="@color/link"
+                    aztec:linkColor="?attr/colorPrimary"
                     aztec:linkUnderline="true"
                     aztec:quoteBackground="@color/quote_background"
                     aztec:quoteColor="@color/quote"
                     aztec:quoteMargin="@dimen/quote_margin"
                     aztec:quotePadding="@dimen/quote_padding"
                     aztec:quoteWidth="@dimen/quote_width"
                     aztec:textColor="@color/text"
-                    aztec:textColorHint="@color/hint_text" >
-                </org.wordpress.aztec.AztecText>
+                    aztec:textColorHint="@color/hint_text" />
 
                 <org.wordpress.aztec.source.SourceViewEditText
                     android:id="@+id/source"
+                    android:layout_width="match_parent"
+                    android:layout_height="match_parent"
+                    android:background="@null"
+                    android:fontFamily="monospace"
                     android:gravity="top|start"
+                    android:imeOptions="flagNoExtractUi"
                     android:inputType="textNoSuggestions|textMultiLine"
-                    android:layout_height="match_parent"
-                    android:layout_width="match_parent"
-                    android:paddingTop="16dp"
-                    android:paddingLeft="16dp"
                     android:paddingStart="16dp"
-                    android:paddingRight="16dp"
+                    android:paddingLeft="16dp"
+                    android:paddingTop="16dp"
```

Comment:
Maybe it would be easier if we set all margins with `android:padding="16dp"` ?

Reply:
Yep, good call 👍